src/sentry/migrations/0522_migrate_discover_savedquery_worldmaps.py

```diff
@@ -4,13 +4,13 @@
 from django.db.backends.base.schema import BaseDatabaseSchemaEditor
 from django.db.migrations.state import StateApps
 
-from sentry.discover.models import DiscoverSavedQuery
 from sentry.new_migrations.migrations import CheckedMigration
 
 
 def migrate_savedquery_worldmap_display_to_totalPeriod(
     apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
 ) -> None:
+    DiscoverSavedQuery = apps.get_model("sentry", "DiscoverSavedQuery")
```

Comment:
migration tests were failing because of a direct import here. is it safe to retro fix a migration like this?

Reply:
Yeah that's ok, if we don't fix this then this migration will fail